csm/conf/etc/cron.daily/es_logrotate-virtual.cron

```diff
@@ -0,0 +1,19 @@
+# CORTX-CSM: CORTX Management web and CLI interface.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
```

Comment:
Should not duplicate files.
The files can be modified using sed command and this can be done during post install phase of provisioner's setup.py framework.

Reply:
The patch has already been landed onto cortx1.0-dev (https://github.com/Seagate/cortx-manager/pull/104) without any notes.
Shall I do the changes anyway?

Reply:
Yes Please.

Reply:
Please do it only for main branch.

Reply:
Done.